src/Meilisearch/Builder/CompositeFilterBuilder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\Meilisearch\Builder;
+
+final class CompositeFilterBuilder implements FilterBuilderInterface
```

Comment:
Rather use the `Setono\CompositeCompilerPass\CompositeService` that's used in several places in the plugin. Here are the docs: https://github.com/Setono/composite-compiler-pass